apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -127,3 +131,13 @@ export default function PokemonLink({
   );
 }
 ```
+
+import NavigationButtons from '../../../components/navigation-buttons'
+
+<NavigationButtons
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space after the opening parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `{`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChAyOGE0NWRkMjI3MTQ0NWVjEgxMYW5ndWFnZVRvb2w= -->